run_dir/static/js/pricing_update.js

```diff
@@ -816,104 +715,104 @@ app.component('components', {
-      this.tooltip = new bootstrap.Tooltip(this.$el)
-  },
-  methods: {
-    updateQuantity(new_val) {
-        if (this.type == 'Alternative') {
-          this.$root.all_products[this.product_id]['Alternative Components'][this.component_id]['quantity'] = new_val
-        } else {
-          this.$root.all_products[this.product_id]['Components'][this.component_id]['quantity'] = new_val
-        }
-    },
-    addComponent(event) {
-        if (event) {
-          console.log(this.product_id)
-          this.$root.addProductComponent(this.product_id, this.component_id, this.type)
-        }
     },
-    removeComponent(event) {
-        /* This took me some time to figure out - the problem is that the bootstrap tooltip would
-        remain open forever if it's not actively removed like this */
-        if (event) {
-          tooltip = bootstrap.Tooltip.getInstance(this.$el)
-          tooltip_el = this.$el
-          that = this
-          function whenHidden() {
-              tooltip_el.removeEventListener('hidden.bs.tooltip', whenHidden)
-              that.$root.removeProductComponent(that.product_id, that.component_id, that.type)
-          }
-          /* Wait until tooltip is actually hidden before removing the component */
-          tooltip_el.addEventListener('hidden.bs.tooltip', whenHidden)
-          tooltip.hide()
-        }
-    }
-  },
-  template: /*html*/`
-    <tr data-toggle="tooltip" data-placement="top" :data-original-title="tooltip_html" data-animation=false data-html=true>
-      <th scope="row">{{component_id}}</th>
-      <td>{{component['Product name']}}</td>
-      <td v-if="added">
-        <!-- I was for some reason unable to solve this with regular v-model... -->
-        <input class="form-control" type="number" min=1 :value="component_data['quantity']" @input="updateQuantity($event.target.value)"/>
-      </td>
-      <td class="pl-3 pt-2" v-if="added">
-        <a class="mr-2" href="#" @click="this.removeComponent">
-          <i class="far fa-times-square fa-lg text-danger"></i>
-        </a>
-      </td>
-      <td class="pl-3 pt-2" v-if="!added">
-        <a class="mr-2" href="#" @click="this.addComponent">
-          <i class="far fa-plus-square fa-lg text-success"></i>
-        </a>
-      </td>
+    template: /*html*/`
+      <tr data-toggle="tooltip" data-placement="top" :data-original-title="tooltip_html" data-animation=false data-html=true>
+        <th scope="row">{{component_id}}</th>
+        <td>{{component['Product name']}}</td>
+        <td v-if="added">
+          <!-- I was for some reason unable to solve this with regular v-model... -->
+          <input class="form-control" type="number" min=1 :value="component_data['quantity']" @input="updateQuantity($event.target.value)"/>
+        </td>
+        <td class="pl-3 pt-2" v-if="added">
+          <a class="mr-2" href="#" @click="this.removeComponent">
+            <i class="far fa-times-square fa-lg text-danger"></i>
```

Comment:
should the class be `fa` instead of `far`?

Reply:
Nope